docs/en/04_Changelogs/3.7.0.md

```diff
@@ -0,0 +1,26 @@
+# 3.7.0
+
+### Versioned cache segmentation
+
+The cache API now maintains separate cache pools for each versioned stage. This prevents users from caching draft data and then exposing it on the live stage.
+
+```php
+// Before:
+$cache = SS_Cache::factory('myapp');
+Versioned::set_reading_mode('Stage.Live');
```

Comment:
Shouldn't the reading mode be the other way around, first `Stage.Stage`, then `Stage.Live`?